src/coffee/react-bootstrap-switch.coffee

```diff
@@ -35,6 +35,7 @@ module.exports = React.createClass
 
   componentWillReceiveProps: (nextProps) ->
     this.value(nextProps.state)
+    this.disabled(nextProps.disabled) if nextProps.disabled === true or nextProps.disabled === false
```

Comment:
I think this is the right idea, but causes build errors. Plus, when this is compiled into JS dist, it replaces the return statement with the disabled function. I think you should inverse the order in which you are calling these methods and get rid of the if conditions--I think the if conditions are pointless, in my opinion as it will always be either true or false.

The reason for all this, I think, is we ultimately want to call the disabled function before returning the value method. At least, when I modified your code it all seemed to work for me :)

So the resulting code here that ended up working for me was:
```
componentWillReceiveProps: (nextProps) ->
 +   this.disabled(nextProps.disabled)
      this.value(nextProps.state)
```